use-cases/assembler_metal/scripts/src/run_jcl.ts

```diff
@@ -0,0 +1,93 @@
+import { SubmitJobs, IJob, GetJobs, DownloadJobs, IDownloadAllSpoolContentParms } from "@zowe/cli";
+import { ISession, SessConstants, Session } from "@zowe/imperative";
+import { readFileSync, existsSync, mkdirSync } from "fs";
+import * as path from "path";
+
+const properties = require("../../config/local.json");
```

Comment:
Should `import * as config from "config"` be used to load config here, same as in "genjcl.ts"? Or is the `config` package unnecessary above? Curious why the config is being loaded differently between the two scripts.

Reply:
It was being done with require here because it added the variable names and typings from the config into intellisense, whereas config did not. I can switch it back if need be.

Reply:
Maybe the config import in the other file could be removed and replaced with `require`? If `require` works equally as well or better, this could perhaps eliminate a dependency.

Reply:
Done